### PR TITLE
new: usr: #17: add support for specifying cluster type

### PR DESCRIPTION
**Pull request by jordanw-bq**

### CODE REVIEW

File: rubix_admin/installer.py [hunk 1]
```diff
@@ -79,19 +83,11 @@ def _rpm_install(cls, args):
...
-        sudo("mkdir -p /etc/presto/conf/catalog")
+    def _rubix_op(cls, args, is_master):
+        sudo("chmod +x /usr/lib/rubix/bin/configure-*.sh")
+        sudo("/usr/lib/rubix/bin/configure-rubix.sh")
 
-        sudo("chmod +x /usr/lib/rubix/bin/configure.sh")
-        sudo("/usr/lib/rubix/bin/configure.sh")
-        sudo("restart presto-server")
+        cluster_type = args.cluster_type
+        if not (is_master is False and cluster_type == "spark"):
```
Comment (abhishekdas99) [inline rubix_admin/installer.py]: Can you change this to separate spark and presto ?

**Comment by abhishekdas99** [comment]
Is it possible to pass on a config file from the user ? Think of a scenario where user wants to start rubix daemons with some specific rubix specific configurations

File: rubix_admin/installer.py (hunk 1 above)
Comment (jordanw-bq) [reply rubix_admin/installer.py]: Updated to improve clarity

**Comment by jordanw-bq** [comment]
@abhishekdas99 I will be addressing custom RubiX configurations as a separate issue.

**Pull request merged**